From d9ddd961094e59ede6658f1776f88f259dc4e45d Mon Sep 17 00:00:00 2001
From: Geert Janssens <geert@kobaltwit.be>
Date: Tue, 30 Apr 2019 11:09:44 +0200
Subject: [PATCH] FindSWIG - allow to provide search prefix on Windows

The default search paths won't find SWIG on windows, so we need a way to
tell cmake where to look. The NO_CMAKE_FIND_ROOT_PATH prevents us
from doing so.
---
 Modules/FindSWIG.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Modules/FindSWIG.cmake b/Modules/FindSWIG.cmake
index fc0ed009da..78592c4ea3 100644
--- a/Modules/FindSWIG.cmake
+++ b/Modules/FindSWIG.cmake
@@ -42,7 +42,7 @@ if(SWIG_EXECUTABLE)
     endif()
   else()
     string(REGEX REPLACE "[\n\r]+" ";" SWIG_swiglib_output ${SWIG_swiglib_output})
-    find_path(SWIG_DIR swig.swg PATHS ${SWIG_swiglib_output} NO_CMAKE_FIND_ROOT_PATH)
+    find_path(SWIG_DIR swig.swg PATHS ${SWIG_swiglib_output})
     if(SWIG_DIR)
       set(SWIG_USE_FILE ${CMAKE_CURRENT_LIST_DIR}/UseSWIG.cmake)
       execute_process(COMMAND ${SWIG_EXECUTABLE} -version
-- 
2.20.1

